app/src/main/java/com/example/allowancetracker/data/Converters.kt

```diff
@@ -13,4 +13,11 @@ class Converters {
     fun dateToTimestamp(date: Date?): Long? {
         return date?.time
     }
+
+    @TypeConverter
+    fun toPurchaseType(purchaseType: PurchaseType): Int = purchaseType.ordinal
```

Comment:
what is an ordinal? :]

Reply:
Ordinal is a not-so-clear 👀 built in name for the `Int` that the enum value converts to

Like:
`enum Color{
   Red,     // Ordinal 0
   Green, // Ordinal 1
   Blue    //Ordinal 2
}

`

Reply:
oooo okay thank you! :D